docs/google.rst

```diff
@@ -64,7 +64,7 @@ The support for Google Batch requires Nextflow version ``22.07.1-edge`` or later
 installed make sure to update to the latest `edge` release using these commands::
 
     export NXF_EDGE=1
-    nextflow -self-update
+    nextflow self-update
```

Comment:
They are both valid

Reply:
Huh, I've never seen it without the hyphen 😅